### PR TITLE
Introduce support for Infinity

### PR DESCRIPTION
**Pull request by vthibault**
This adds support for `Infinity` value (previously converted to `null`).

_I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner._

### CODE REVIEW

**Comment by papb** [comment]
How about `-Infinity`?

**Comment by vthibault** [comment]
Nice catch ! Just implemented on 4599ec6.

File: test/unit/serialize.js [hunk 1]
```diff
@@ -335,16 +335,18 @@ describe('serialize( obj )', function () {
...
+            expect(serialize(m)).to.be.a('string').equal('new Map([["a",123],[new RegExp(".*", ""),456],[Infinity,322]])');
+            expect(serialize({t: [m]})).to.be.a('string').equal('{"t":[new Map([["a",123],[new RegExp(".*", ""),456],[Infinity,322]])]}');
         });
 
         it('should deserialize a map', function () {
             var m = eval(serialize(new Map([
                 ['a', 123],
-                [null, 456]
+                [null, 456],
+                [Infinity,322]
```
Comment (okuryu) [inline test/unit/serialize.js]: I think it should be `789` to match the above regularity.

Comment (vthibault) [reply test/unit/serialize.js]: I failed to notice the sequential suit. Here you are: 6ac6afe

**Comment by okuryu** [comment]
Published v3.0.0. Thanks!

**Pull request merged**